src/main/java/com/ft/methodearticleinternalcomponentsmapper/transformation/InternalComponentsMapper.java

```diff
@@ -190,7 +186,7 @@ private String validateBody(final TransformationMode mode,
             return EMPTY_VALIDATED_BODY;
         }
 
-        if (Type.CONTENT_PACKAGE.equals(type)) {
```

Comment:
I would change this to type.equals(CONTENT_PACKAGE) and I would do the same for TransformationMode.PREVIEW.equals(mode).